files/system/usr/share/ublue-os/just/70-secureblue.just

```diff
@@ -8,6 +8,13 @@ set-kargs-hardening:
         IAEMU_NO='--append-if-missing="ia32_emulation=0"'
         echo "Disabling 32-bit support, for the next boot."
     fi
+    read -p "Do you want to disable Simultaneous Multithreading (SMT) / Hyperthreading? (This can cause a reduction in the performance of certain tasks in favor of security, note that your hardware may already have SMT disabled to mitigate a known vulnerability) [y/N]: " YES
+    if [[ "$YES" == [Yy]* ]]; then
+        NOSMT_YES='--append-if-missing="nosmt=force"'
+        echo "SMT/Hyperthreading disabled, requires reboot to take effect."
```

Comment:
We can remove "requires reboot to take effect" since that's true for any karg or any rpm-ostree operation

Reply:
Also to be consistent with the other statements:

"Force disabling SMT/Hyperthreading"

"Not force disabling SMT/Hyperthreading"